### PR TITLE
2.6.0: Some Improvements

### PR DESCRIPTION
**Pull request by Kikobeats**
## Features
- [x] Better post/tag view.
- [x] Emojis style based on Twitter.
- [x] Code Highlights based on Gist.
- [x] Blockquotes style.
## To fix
- [x] Improve mobile post view.
## Demo
- http://kikobeats.com.
- http://kikobeats.com/tag/craftsmanship.
- http://kikobeats.com/styleguide.

### CODE REVIEW

**Comment by Kikobeats** [comment]
feedback is appreciated!

@foorb @kutyel @RReverser  @Mooash

:-)

**Comment by foorb** [comment]
Hey,
I don't like the mobile view. Because of the date, the title breaks in many lines.
The date below the title would be much nicer. But thats my personal view.

screenshot_2015-11-23-15-39-52

**Comment by jloh** [comment]
Yeah, I also agree with @foorb. It seems like a lot of wasted whitespace. That said, the new post format looks great! Amazing work!

**Comment by Kikobeats** [comment]
It's true. @foorb thank your for the refactor idea.

I created a checklist of things to resolve to finally release the version, feel free to say more things :)

**Comment by Kikobeats** [comment]
@foorb @Mooash Can you check mobile version right now in my blog?

screen shot 2015-12-03 at 08 25 42

**Comment by foorb** [comment]
Great

File: index.hbs [hunk 1]
```diff
@@ -1,17 +1,14 @@
 {{!< default}}
-<ol class="posts">
+
+<h1 style="text-align: center;">Writtings.</h1>
+<ol id="posts-list">
   {{#foreach posts}}
   <li>
-    <a href="{{url}}" title="link to {{{title}}}">
-      <h2>{{{title}}}</h2>
-      <p class="post description">{{excerpt}}&hellip;</p>
```
Comment (se7entyse7en) [inline index.hbs:7]: is it possible in some way to reenable the posts description?

Comment (Kikobeats) [inline index.hbs:7]: What's your point for do it?

Most of the time description is not useful and looks like generic text IMHO

**Pull request merged**